lookup_plugins/parse_mt_report.py

```
@@ -0,0 +1,626 @@
+        return write_to_json(html_content, output_file_path, type, return_string=return_string)
+    elif type in ["opt"]:
+        return write_opt(html_content, output_file_path, include_titles, return_string=return_string)
+
+    raise ValueError('Incorrect type passed. Allowed value: "orders" OR "deals" OR "header" OR "opt".')
+
+class LookupModule(LookupBase):
+    # Expecting the same list of arguments as the "main" function.
+    def run(self, terms, variables=None, **kwargs):
+        if len(terms) < 3:
+            raise AnsibleError('parse_mt_report requires exactly 3 parameters to be passed:\n1) input file path\n2) type of file: "orders"|"deals"|"header"|"opt"\n3) include titles?')
+        (input_file_path, type, include_titles) = terms
```

Ruff (0.15.12)

[error] Variable `type` is shadowing a Python builtin

(A001)